storage.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+from pkmntypes import Pokemon
+from typing import Union
+from motor.motor_asyncio import AsyncIOMotorClient, AsyncIOMotorClientSession, AsyncIOMotorCollection, AsyncIOMotorDatabase
+import logging
+
+log = logging.getLogger(__name__)
+client: AsyncIOMotorClient = AsyncIOMotorClient('mongodb://db/brock')
+db: AsyncIOMotorDatabase = client.brock
+
+
+def _set_client(new_client: AsyncIOMotorClient):
+	"""Manually set the mongodb client that `storage` should use. Only to be used in tests."""
+
+	global client, db
+	client = new_client
+	db = client.brock_test
+
+
+def pokemon() -> AsyncIOMotorCollection:
+	"""Get the Pokemon collection to be able to interact with the database."""
+	return db.pokemon
+
+
+async def save_object(obj: Union[Pokemon], session: AsyncIOMotorClientSession = None):
+	"""Save the given object to the database. If `_id` is not set, a new entry will be created. If `_id` is set, it will try to update the document with that ID.
```

Comment:
```suggestion
	"""Save the given object to the database. If `_id` is not set, a new entry will be created and the object's `_id` will be updated. If `_id` is set, it will try to update the document with that ID.
```